src/styles/blocks/price-box.scss

```diff
@@ -0,0 +1,15 @@
+.price-box {
+  @include displayFlex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
```

Comment:
its default, remove it plz

Reply:
When i remove it the size change, probably because I use H3 tag

Reply:
My solution is make h3 default font size 12px.